### ISSUE TITLE
Can't get wizard running on windows

### ISSUE THREAD

**Issue by melund** [scopatz/xonsh#640]
I just tried to delete my `.xonshrc` to try the Wizard. Sorry, I haven't had time before.

I doesn't seem to start for me...

![image](https://cloud.githubusercontent.com/assets/1038978/12495806/1a2bd7d0-c092-11e5-8cce-a164e0e5a069.png)

Any tricks that I should know about.

**Comment by scopatz** [comment]
Did you put a space after yes?

**Comment by scopatz** [comment]
Ahhh or maybe it is a '\r'

**Comment by scopatz** [comment]
Also you can always try running this manually via the `xonfig wizard` command.

**Comment by melund** [comment]
Doesn't seem to do anything

![image](https://cloud.githubusercontent.com/assets/1038978/12505845/8d5e02b6-c0ea-11e5-8193-0c580881c188.png)

I am running from master with `python setup.py develop`

**Comment by scopatz** [comment]
Can you please comment out the try-except in the xonsh.xonfig._wizard() function so that we can see what the error is please?

**Comment by melund** [comment]
Ahh.. Now I see the problem: 
`UnicodeEncodeError: 'charmap' codec can't encode character '\xb8' in position 822: character maps to <undefined>`

It is two character in the fancy ruler that is causing the issues. 
```
HR = "'`-.,¸,.-*¯`-.,¸,.-*¯`-.,¸,.-*¯`-.,¸,.-*¯`-.,¸,.-*¯`-.,¸,.-*¯`-.,¸,.-*'"
```

Removing ¯ and ¸ allow me to start the wizard. There is however a small issue with the coloring in the wellcome screen. Do you have the that issue as well. 

![image](https://cloud.githubusercontent.com/assets/1038978/12522648/18dcda4e-c152-11e5-9f3c-4180d8710e5e.png)

**Comment by melund** [comment]
This was fixed by #642
